internal/getproviders/types.go

```diff
@@ -137,6 +137,11 @@ func (p Platform) LessThan(other Platform) bool {
 // ParsePlatform parses a string representation of a platform, like
 // "linux_amd64", or returns an error if the string is not valid.
 func ParsePlatform(str string) (Platform, error) {
+	parts := strings.Split(str, "_")
+	if len(parts) != 2 {
+		return Platform{}, fmt.Errorf("must be two words separated by an underscore")
+	}
+
```

Comment:
I think this check replaces the next one

Reply:
it does not, that's why I added it  and the supporting tests -- those and the command test I wrote failed, initially.

Reply:
It only checks that there is _an_ underscore, so you can have multiples.

Reply:
Sorry, I mean your new one supersedes the following check, which manually splits the part in 2. Deleting the next block and using `os, arch := parts[0], parts[1]` should work exactly the same.

Reply:
That's true. I almost made that change, but decided to leave the rest  as-is. I'll go do that instead. Thanks!